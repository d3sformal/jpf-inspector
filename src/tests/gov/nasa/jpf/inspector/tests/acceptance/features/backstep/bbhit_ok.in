# All bbhit's in this file work well.


cr bp local_write = a
run && wait

so && wait

bbhit && wait

# back at a = 5

so && wait
so && wait
so && wait

del bp 1
back_breakpoint_hit && wait

# back at a = 5

thpc
run && wait


