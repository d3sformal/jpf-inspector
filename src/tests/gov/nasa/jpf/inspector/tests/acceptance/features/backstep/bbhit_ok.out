INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 local_write=a]
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/sut/MultiTransitionSinglePath.java:9 (iconst_1), source: int transitionStart = Verify.getInt(1, 2);
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/sut/MultiTransitionSinglePath.java:8 (istore_1), source: int a = 5;
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/sut/MultiTransitionSinglePath.java:9 (iconst_1), source: int transitionStart = Verify.getInt(1, 2);
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/sut/MultiTransitionSinglePath.java:10 (iload_1), source: int b = a + 4;
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/sut/MultiTransitionSinglePath.java:11 (iload_3), source: int c = b - 8;
Breakpoint with ID 1 successfully deleted.
INFO: SuT is stopped
	SuT (thread 0) will now execute gov/nasa/jpf/inspector/tests/acceptance/sut/MultiTransitionSinglePath.java:8 (istore_1), source: int a = 5;
0 : gov/nasa/jpf/inspector/tests/acceptance/sut/MultiTransitionSinglePath.java:8:    int a = 5;
	gov.nasa.jpf.inspector.tests.acceptance.sut.MultiTransitionSinglePath:main:1:istore_1
INFO: JPF Terminating